src/theme/material/date-input.m.css

```diff
@@ -0,0 +1,43 @@
+@import './variables.css';
+
+.root {
+}
+
+.input {
+}
+
+.root .toggleCalendarButton {
```

Comment:
may have been easier to simply use a `button` rather than a widget `Button`

Reply:
Good call.